src/Composer/ComposerWindow.vala

```diff
@@ -19,18 +19,27 @@
  */
 
 public class Mail.ComposerWindow : Gtk.ApplicationWindow {
+    public ComposeRequest template { get; construct; }
```

Comment:
If possible, I think it helps to make variable names include the class name. So for example, `compose_request` or `template_composerequest` makes it more clear that this is an instance of `ComposeRequest`. This really helps when someone else later down the line is trying to debug the code and wondering what kind of object `template` is